File: Restaurant-Api-SpringBoot/src/main/java/br/com/felipetvsouza/response/Address.java
package br.com.felipetvsouza.response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Builder
@Getter
@Setter
public class Address {

    private String building;

    private String street;

    private String zipcode;

    private List<Double> coord;

}

File: Restaurant-Api-NetCore/src/Br.Com.Restaurant.Business/Models/Address.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Br.Com.Restaurant.Business.Models
{
    public class Address
    {
        public string building { get; set; }

        public string street { get; set; }

        public string zipcode { get; set; }

        public List<Double> coord { get; set; }
    }
}
